aux/test/test_eigen.cxx

```diff
@@ -0,0 +1,91 @@
+#include "WireCellAux/Util.h"
+#include "WireCellUtil/Testing.h"
+
+#include <iostream>
+#include <chrono>
+
+typedef std::chrono::high_resolution_clock Clock;
+
+using namespace WireCell;
+
+namespace
+{
+template <typename ElementType>
+std::string dump(const ITensor::pointer iten)
+{
+    std::stringstream ss;
+    ss << "ITensor: \n";
+    Json::FastWriter jwriter;
+    for (size_t i1 = 0; i1 < iten->shape()[1]; ++i1)
+    {
+        for (size_t i0 = 0; i0 < iten->shape()[0]; ++i0)
+        {
+            size_t ind = i1 * iten->shape()[0] + i0;
+            auto data = (const ElementType *)iten->data();
+            ss << data[ind] << " ";
+        }
+        ss << "\n";
+    }
+
+    return ss.str();
+}
+} // namespace
+
+void test_translation() {
+    typedef float TestType;
+
+    // Eigen::Array<TestType, Eigen::Dynamic, Eigen::Dynamic> iarr =
+    // Eigen::Array<TestType, Eigen::Dynamic, Eigen::Dynamic>::Random(2, 3);
+    Eigen::Array<TestType, 2, 3> iarr;
+    iarr << 1, 2, 3,
+            4, 5, 6;
+    std::cout << iarr << "\n";
+
+    auto iten = Aux::eigen_array_to_itensor<TestType>(iarr);
```

Comment:
Besides `cout` lines for a one time "by eye" test, it's good to litter the code with asserts.  `WireCellUtil/Testing.h` provides `Assert()` and `AssertMsg()` to provide some value-add beyond standard `assert()`.

Reply:
There was one, forgot to uncomment before pushing. Fixed.